core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -631,6 +631,29 @@ public static int nextGreaterValueToTarget(int currentIndex,
     return integers;
   }
 
+  /**
+   * Convert path with start and end string
+   * Convert / . \ : space to _
+   * remove duplicate _
+   *
+   * @param path   store path
+   * @param prefix prefix string
+   * @param suffix suffix string
+   * @return converted string
+   */
+  public static String convertPath(String path, String prefix, String suffix) {
```

Comment:
please add UT for this func

Reply:
remove this one and change the table name to UnknownTable+time